docs/l10n.md

```diff
@@ -0,0 +1,39 @@
+# Translating Trustroots (localization)
```

Comment:
I wrote this document to my best limited ability. Improvements and complete rewrites are very welcome!